haxepunk/Screen.hx

```diff
@@ -324,23 +255,6 @@ class Screen
 	function get_mouseY():Int return Std.int(_sprite.mouseY);
 
 	/**
-	 * Captures the current screen as an Image object.
-	 * Will only work in buffer rendermode (flash or html5).
-	 * @return	A new Image object.
-	 */
-	public function capture():Image
```

Comment:
This wasn't possible with drawTiles but now that we have our own renderer we should revisit. Maybe keep this method for now, update the comment and have it throw "Not yet implemented"?